wiki/Beatmap_ranking_procedure/id.md

```diff
@@ -1,52 +1,43 @@
+Beatmap yang memiliki 2 nominasi dari masing-masing mode permainan yang ada di beatmapset tersebut akan dipindahkan ke kategori beatmap [Qualified](/wiki/Beatmap/Category#qualified) selama 7 hari. Qualified beatmap memiliki scoreboard tapi tidak akan menghasilkan [performance points](/wiki/Performance_Points). Ketika beatmap berada di Qualified setidaknya selama 7 hari, [antrian ranking](Ranking_queue) dapat memindahkannya ke [Ranked](#ranked).
 
-Ketika beatmap terdiskualifikasi, posisinya dari antrian ranking tetap berada di antrian itu. Jika ternyata bisa masuk kembali ke dalam qualified, beatmap tersebut akan mendapatkan posisi yang sama dalam antrian ranking. Contoh, jika beatmap telah qualified selama 2 hari, lalu dinyatakan diskualifikasi dan ternyata masuk kembali ke dalam qualified, beatmap tersebut akan berada dalam Qualified dengan perkiraan 5 hari. Jika posisi antrian yang tersimpan jumlahnya kurang dari 24 jam dari Ranked, antrian menjadi ranked akan diatur ulang menjadi 24 jam setelah beatmap tersebut Qualified.
+Kategori ini ada untuk memberikan tanggapan-tanggapan dari Komunitas osu! dengan harapan potensi masalah dapat ditemukan. Masalah tersebut dapat dilaporkan kepada anggota BN atau NAT pada halaman diskusi beatmap. Berbeda dengan Pending beatmap, Qualified beatmap tidak dapat diperbarui oleh pembuat. Oleh karena itu, umpan balik atau kritik dapat diterapkan setelah meminta sebuah [pengulangan nominasi](#pengulangan-nominasi).
 
 ## Pengulangan nominasi
 
 Pengulangan nominasi *(nomination reset)* membantu memastikan bahwa modder, BN dan anggota NAT memeriksa versi terbaru dari beatmapset sebelum memasuki antrian ranking.
 
-Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) ketika alasannya beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset memenuhi syarat ketika nominasinya disetel ulang, beatmapset tersebut akan didiskualifikasi, dan terhapus dari antrian ranking
+Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) apabila beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset sedang berada di Qualified ketika nominasinya disetel ulang, beatmapset tersebut akan didiskualifikasi dan dihapus dari antrian ranking.
```

Comment:
```suggestion
Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN maupun NAT menemukan sebuah masalah di dalam beatmapset yang telah dinominasi. Beatmapset juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) apabila beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset sedang berada di Qualified ketika nominasinya disetel ulang, beatmapset tersebut akan didiskualifikasi dan dihapus dari antrian ranking.
```

Reply:
the repeating `atau` seems ambiguous here
yeah don't think the word has to shorten here